machineconfiguration/v1/controllerconfig.crd.yaml

```diff
@@ -0,0 +1,452 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: controllerconfigs.machineconfiguration.openshift.io
```

Comment:
i'm not sure this should be part of the openshift/api since this looks like an machine-config-operator internal API..

Reply:
they are in the same group at the moment (`machineconfiguration.openshift.io`). Should we move them out of that group into an MCO-internal group? Are there conventions around this?